package.json

```diff
@@ -72,7 +72,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^5.1.1",
     "eslint-plugin-standard": "^5.0.0",
-    "graphql": "^16.0.0",
+    "graphql": "^17.0.0-alpha.2",
```

Comment:
This could be also changed back to 16 before merging, as 17 is nevertheless still alpha. We will keep support in test runs, though.